src/Workspaces/Core/Portable/CodeGeneration/AbstractFlagsEnumGenerator.cs

```diff
@@ -15,10 +13,10 @@ namespace Microsoft.CodeAnalysis.CodeGeneration
     internal abstract class AbstractFlagsEnumGenerator : IComparer<(IFieldSymbol field, ulong value)>
     {
         protected abstract SyntaxGenerator GetSyntaxGenerator();
-        protected abstract SyntaxNode CreateExplicitlyCastedLiteralValue(INamedTypeSymbol enumType, SpecialType underlyingSpecialType, object constantValue);
+        protected abstract SyntaxNode CreateExplicitlyCastedLiteralValue(INamedTypeSymbol enumType, SpecialType underlyingSpecialType, object? constantValue);
         protected abstract bool IsValidName(INamedTypeSymbol enumType, string name);
 
-        internal SyntaxNode CreateEnumConstantValue(INamedTypeSymbol enumType, object constantValue)
+        internal SyntaxNode? CreateEnumConstantValue(INamedTypeSymbol enumType, object constantValue)
```

Comment:
The idea of a `Create` method returning `null` really subverts my expectations hence it's really nice to see nullable annotations here to help with consumption.

Reply:
Agreed.  This was moved down from IDE code.  And, IMO, should be called TryCreate.